packages/host/app/components/operator-mode/create-file-modal.gts

```diff
@@ -765,6 +783,57 @@ export class ${className} extends ${exportName} {
       this.saveError = `Error creating card instance: ${e.message}`;
     }
   });
+
+  private createBoxelSpecInstance = restartableTask(async () => {
```

Comment:
i think its probably time to split the different files that can be created into different modules. right now this module is doing _a lot_. perhaps we can use a strategy pattern to figure out how to create the right type of file.

Reply:
it could be a simple as having a module that exports an async function that is the meat of this task, and then setting up a map of file types to functions to make the file type and then just executing the async function that makes the file.